index.html

```diff
@@ -22,6 +22,7 @@
 
     <header class="intro">
        <!-- image code here -->
+       <img src="img/main-header.png" alt="">
```

Comment:
if you give a width of 100% to this image the image will always expand to the total width of the parent div making it resizable